addons/account/controllers/portal.py

```diff
@@ -11,7 +11,7 @@ class PortalAccount(CustomerPortal):
 
     def _prepare_portal_layout_values(self):
         values = super(PortalAccount, self)._prepare_portal_layout_values()
-        invoice_count = request.env['account.invoice'].search_count([])
+        invoice_count = request.env['account.move'].search_count([])
```

Comment:
@smetl 
should it not be with the domain type in ('out_invoice', 'in_invoice', 'out_refund', 'in_refund')?

Reply:
@csnauwaert Indeed